assets/src/edit-story/components/canvas/editElement.js

```diff
@@ -47,6 +47,7 @@ function EditElement({ element }) {
     actions: { getBox },
   } = useUnits();
 
+  //eslint-disable-next-line @wordpress/no-unused-vars-before-return
```

Comment:
I'm wondering why doesn't it know about the usage? Does it not expand JSX before running these rules?

Reply:
https://github.com/WordPress/gutenberg/pull/21358 has a pending fix with details.